linkis-commons/linkis-common/src/main/java/org/apache/linkis/common/io/Fs.java

```diff
@@ -52,4 +52,8 @@
 
     public abstract boolean renameTo(FsPath oldDest, FsPath newDest) throws IOException;
 
+    public abstract boolean mkdir(FsPath dest) throws IOException;
```

Comment:
Hello, the code implemented by the method is not uploaded

Reply:
interface Fs in module linkis-common have a direct implement abstract class FileSystem in module linkis-storage. The method add into Fs have already IN FileSystem  before this PR

Reply:
I have remove the the abstract method mkdir\mkdirs from abstract class FileSystem and lift them to interface